activerecord/lib/active_record/connection_adapters/postgresql/schema_dumper.rb

```diff
@@ -4,7 +4,27 @@ module ActiveRecord
   module ConnectionAdapters
     module PostgreSQL
       class SchemaDumper < ConnectionAdapters::SchemaDumper # :nodoc:
+        def dump(stream)
+          header(stream)
+          extensions(stream)
```

Comment:
I think duplicating `dump` may introduce bugs in the future. How about keeping `dump` unchanged in the super class and implementing `extensions` as an empty method on `ConnectionAdapters::SchemaDumper` and on this class with the current implementation?

Reply:
Great idea. Agreed. Will update the pull request.